typespec-extension/src/code-model-builder.ts

```diff
@@ -667,7 +672,28 @@ export class CodeModelBuilder {
       this.trace(`Operation '${operation.name}' is temporary skipped, as it is an overloaded operation`);
       return true;
     }
-    return false;
+    return !this.existsAtCurrentVersion(operation) || false;
```

Comment:
Seems easier to understand by putting it at top, e.g.

```
if (!this.existsAtCurrentVersion(operation)) {
  return true;
}
```

Reply:
fixed